### ISSUE TITLE
Rate limits not properly handled by WebhookClient

### ISSUE THREAD

**Issue by RighteousPotato** [discordjs/discord.js#2724]
**Please describe the problem you are having in as much detail as possible:**
An UnhandledPromiseRejection is thrown when hitting rate limits on a webhook

error:   Uncaught Promise Rejection:
TypeError: this.client.listenerCount is not a function
    at item.request.gen.end (\node_modules\discord.js\src\client\rest\RequestHandlers\Sequential.js:87:29)
    at then (\node_modules\snekfetch\src\index.js:214:21)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)

**Include a reproducible code sample here, if possible:**
```js
const client = new Discord.WebhookClient(id, token);

for(let x=0; x<30; x++){
	client.send(`Message #${x}`);
};
```

**Further details:**

- discord.js version: 11.4.0
- node.js version: 8.9.3
- Operating system: Windows 7 x64
- Priority this issue should have – please be realistic and elaborate if possible: Medium
- [ ] I found this issue while running code on a __user account__
- [ ] I have also tested the issue on latest master, commit hash:

**Comment by RighteousPotato** [comment]
Defining listenerCount fixes it
```js
client.listenerCount = function(){return 0};```

**Comment by Lewdcario** [comment]
This should be fixed in `11.4-dev` now, if you'd like to make sure on there.

**Comment by RighteousPotato** [comment]
Yep looks like that commit fixes it
